README.md

```diff
@@ -1,4 +1,5 @@
-# Azure Code Snippets
-Variety of snippets to use with Microsoft Azure
-
-- [Alert on Azure API Management Failures for a specific API](alert-azure-api-management-failed-requests-per-api)
+# Azure Code Snippets
+Variety of snippets to use with Microsoft Azure
+
+- [Alert on Azure API Management Failures for a specific API](alert-azure-api-management-failed-requests-per-api)
+- [Lookup the IP Address of API Management Feature after deployment](lookup-azure-api-management-instance-ip-address)
```

Comment:
Would suggest to remove `Feature` as I'm not sure what you are referring to, but open for discussion!
```suggestion
- [Lookup the IP Address of Azure API Management after deployment](lookup-azure-api-management-instance-ip-address)
```